client/src/modules/distribution_center/modals/distribution_key.modal.js

```diff
@@ -0,0 +1,77 @@
+angular.module('bhima.controllers')
+  .controller('DistributionKeyModalController', DistributionKeyModalController);
+
+DistributionKeyModalController.$inject = [
+  '$state', 'NotifyService', 'DistributionCenterService', 'PayrollConfigurationService',
+  'ExchangeRateService', 'SessionService', '$uibModalInstance', 'PatientInvoiceService',
+  'FeeCenterService', 'util', 'settings', '$translate',
+];
+
+function DistributionKeyModalController(
+  $state, Notify, DistributionCenter, Configuration,
+  Exchange, Session, ModalInstance, Invoices, FeeCenters, util, settings, $translate,
+) {
```

Comment:
These parameters must be removed : Configuration, Exchange, Invoices and util because they are not used.